<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 700 19pt &quot;Ubuntu&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="voteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
    background-color: rgb(94, 143, 225); /* Button background */
    color: white; /* White text */
    border: none;
    border-radius: 10px; /* Rounded corners */
    padding: 10px 20px; /* Padding */
}

QPushButton:hover {
    background-color: rgb(55, 109, 200); /* Darker shade on hover */
}

QPushButton:pressed {
    background-color: rgb(94, 143, 225); /* Background when pressed */
    border: 1px solid #1c5985; /* Subtle border when pressed */
}

QPushButton:disabled {
    background-color: rgb(200, 200, 200); /* Light gray background when disabled */
    color: rgb(150, 150, 150); /* Gray text when disabled */
}
</string>
    </property>
    <property name="text">
     <string>voter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxOptions">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
      background-color: white;
  border: 3px solid black;
       border-radius: 10px;
      padding: 10px;
       font-size: 18px;
      font-weight: bold;

       color: #333;
    }
</string>
    </property>
    <property name="title">
     <string>     list of contestants</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>471</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(36, 31, 49);
  border-radius: 10px;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>341</height>
       </rect>
      </property>
      <widget class="QRadioButton" name="option1">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>249</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Gheraibia Roufaida</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="option2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>150</y>
         <width>261</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Barak Obama</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="option3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>211</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Vladimir Putin</string>
       </property>
      </widget>
      <widget class="QLabel" name="contestantImage1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border-radius: 49%;  /* Creates a perfect circle */
    border: 3px solid black;  /* Optional: Border for visibility */
    padding: 0px;  /* Remove padding */
    background-color:  rgb(0, 0, 0) ;/* Transparent background */
    overflow: hidden;  /* Clips anything outside the circle */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="contestantImage2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border-radius: 49%;  /* Creates a perfect circle */
    border: 3px solid black;  /* Optional: Border for visibility */
    padding: 0px;  /* Remove padding */
    background-color:  rgb(0, 0, 0) ;/* Transparent background */
    overflow: hidden;  /* Clips anything outside the circle */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="contestantImage3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border-radius: 49%;  /* Creates a perfect circle */
    border: 3px solid black;  /* Optional: Border for visibility */
    padding: 0px;  /* Remove padding */
    background-color:  rgb(0, 0, 0) ;/* Transparent background */
    overflow: hidden;  /* Clips anything outside the circle */
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font: 800 21pt &quot;Ubuntu Sans&quot;;
color:    rgb(23, 52, 103)
</string>
    </property>
    <property name="text">
     <string> Your Voice Matters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(94, 143, 225)</string>
    </property>
    <property name="text">
     <string> World president 2047</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
